### DISCUSSION TITLE
running new builds from Azure pipelines on MacOS 11.7 (Big Sur)

### DISCUSSION THREAD

**Discussion by valdisj** [OpenOrienteering/mapper#2288] [Questions & Answers] [answered]
Tried to run downloaded artefact from Azure Pipelines for Mapper, but seems to me, there is a limitation for it to only run starting from 12.xx and on. Is this so? Is there any specific code used so that only newest OS are supported?

**Comment by dg0yt** [comment, marked as answer]
AFAICT there were no intentional changes. However, the "deployment target" must be respected by every component in Mapper. Our heaviest dependency, Qt 5.12, needs at least macOS 10.12. 

Today, these macOS versions were connected to versions of Xcode and SDK which are no longer in use. Supporting anything lower than the version 12 will be best-effor.t (The CI image will need to be updated to 13 soon, but I have one MBA stuck at 12.7 aka Monterey.)

**Reply by dg0yt** [reply]
And even macOS 12 is "unmaintained" today. HW sold six years ago. :disappointed:   
https://en.wikipedia.org/wiki/MacOS_version_history#Releases

**Comment by dg0yt** [comment]
I'm trying to explicitly set the deployment target to 11.7 as part of switching the continuous integration to macOS 13 - at least as long as we are building the 0.9.x family of code.

**Reply by dg0yt** [reply]
This work is merged now: The pipelines are configured for deployment target 11.7 now, and relevant warnings no longer appear during the build.

Please test the result.

**Comment by valdisj** [comment]
Excellent, Kai!
Now everything works fine on MacOS 11.7.
p.s. For everyone else, who reads this, until the official release is out (depends on Kai and other masters), and put on OpenOrienteering website, the most current builds can be downloaded from https://dev.azure.com/OpenOrienteering/Mapper/_build?definitionId=2
